WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -1,12 +1,12 @@
 <% provide(:title, "Registrations for #{@competition.name}") %>
 
 <%= render layout: "nav" do %>
-  <%= wca_table do %>
+  <%= wca_table data: { toggle: :table } do %>
```

Comment:
I thought that's a good idea and even better would be just sortable: boolean, but I had to add more data attributes to customize the table and they probably would differ for each sortable table and the data: solution is still necessary. Also it follows rails pattern to add possibility to pass a data hash to such a tag helper.